package.json

```diff
@@ -24,6 +24,7 @@
     "@emotion/styled": "^11.3.0",
     "@mdx-js/mdx": "^2.0.0",
     "algoliasearch": "^4.12.1",
+    "amplify-cli-core": "3.6.2",
```

Comment:
should this be carat versioned?

Reply:
I originally had this set to `latest`, which would only require a lockfile update when updating docs after a CLI release. It's pinned to be explicit, but I don't see any issues with adding a carat

Reply:
Also, please switch to `^4.0.0`

Reply:
@sobolk I've bumped the version and changed the package name, though unfortunately I am unable to import the help information 
```ts
import { commandsInfo } from '@aws-amplify/amplify-cli-core';
```
![image](https://user-images.githubusercontent.com/5033303/228075330-87639268-67c4-4216-8d76-95c420186a2e.png)

Thankfully using yarn will allow us to still import the transient dependency, so it's not entirely necessary to update the import just yet